__tests__/FunctionTest.js

```diff
@@ -0,0 +1,12 @@
+import 'react-native';
+import React from 'react';
+import Home from '../src/Home';
+
+import renderer from 'react-test-renderer';
+
+it('check function and dtate test case', ()=>{
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247177)
```suggestion
it('check function and dtate test case', () =>{
```